modules/mono/editor/Godot.NET.Sdk/Godot.SourceGenerators/ExtensionMethods.cs

```diff
@@ -212,11 +212,17 @@ public static string FullQualifiedSyntax(this SyntaxNode node, SemanticModel sm)
 
         private static void FullQualifiedSyntax(SyntaxNode node, SemanticModel sm, StringBuilder sb, bool isFirstNode)
         {
-            if (node is NameSyntax ns && (isFirstNode || node.Parent is not MemberAccessExpressionSyntax))
+            if (node is NameSyntax ns)
             {
-                SymbolInfo nameInfo = sm.GetSymbolInfo(ns);
-                sb.Append(nameInfo.Symbol?.ToDisplayString(FullyQualifiedFormatIncludeGlobal) ?? ns.ToString());
-                return;
+                bool isMemberAccess = !isFirstNode && node.Parent is MemberAccessExpressionSyntax;
+                bool isInitializer = isFirstNode && node.Parent is AssignmentExpressionSyntax aes && aes.Parent is InitializerExpressionSyntax;
```

Comment:
```suggestion
                bool isInitializer = isFirstNode && node.Parent is AssignmentExpressionSyntax { Parent: InitializerExpressionSyntax };
```

This can be merged into a single pattern, since we don't use `aes` for anything else later.